### PR TITLE
{bio}[foss/2016b,foss/2018b] GEMMA 0.97 + 0.98.1

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b4a459a83d3fb8a5d650db397ace306b for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f38e6a540603737c7c13fbf4ec7a87e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2434.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b27b4447b45a72a861ffb5b1639a78af for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 1 (2 easyconfigs in this PR)
login20.cluster.bc2.ch - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7102ff95ca34e5b44373c577a6b3844b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2f8cf534c62fbcb9b0f759c8f40f70f2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node2434.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/67dc169f88d15d79500c36a4e9d6dcc6 for a full test report.

**Comment by boegel** [comment]
@pescobar So what's the deal with this? :)

File: easybuild/easyconfigs/g/GEMMA/GEMMA-0.98.1-foss-2018b.eb [hunk 1]
```diff
@@ -23,7 +23,6 @@ builddependencies = [
 
 dependencies = [
     ('GSL', '2.5'),
-    ('zlib', '1.2.11'),
```
Comment (akesandgren) [inline easybuild/easyconfigs/g/GEMMA/GEMMA-0.98.1-foss-2018b.eb]: If GEMMA uses zlib, then zlib should be an explicit dependency and not implicit from the toolchain.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
nic424 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/c99b5c6a34420bd072129f01366526f8 for a full test report.

**Comment by smoors** [comment]
@boegel this needs the workaround for OpenBLAS to pass the tests (which is quite worrisome):
https://github.com/easybuilders/easybuild-easyconfigs/pull/8396

**Comment by smoors** [comment]
Going in, thanks @pescobar!

**Comment by smoors** [comment]
@pescobar hm I missed the last comment of @akesandgren : please re-add the `zlib` dep?

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in this PR)
nic407 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/aa02ed83ee18063d680e29300533ad12 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @pescobar!

**Comment by smoors** [comment]
Going in, thanks @pescobar!

**Pull request merged**